### PR TITLE
Feb API test additions

### PR DESCRIPTION
**Pull request by timstackblock**
Hello please review my additions to the API test. @rafaelcr @zone117x @andresgalante 

I added test coverage for deployed contracts. 
I added a heartbeat test that looks for 404 errors when hitting API endpoints.

I am currently working on adding Github actions and a scheduling cron job and should have this pointed to the test environment and added to CI by next week.

Thank you

### CODE REVIEW

**Comment by timstackblock** [comment]
I think I setup CI correctly only thing is I need the test from this build to double check so I will merge the PR you can still comment on it and I will make all the changes requested I just need this merged to see if my cron job is working

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+account_address=ST3C4YVASFG37P8E6210J9KHZJB4MF8ZRBQFKH41N
+recipient=ST297VG59W96DPGBT13SGD542QE1XS954X4ZSWW2J
+private_key=fd3609e8b9352facf360c950d362afe8f0f108b9041750f54b017efc479efeb001
```
Comment (rafaelcr) [inline .env:3]: Now that you have Github actions in place, you should move this to the repo secrets

File: package.json [hunk 2]
```diff
@@ -10,7 +10,10 @@
     "typescript": "^4.9.4"
   },
   "scripts": {
-    "test": "ts-node ./script/index.ts"
+    "test": "ts-node ./script/index.ts",
+    "triggerAnchorBlockTests": "ts-node ./script/index.ts triggerAnchorBlockTests",
+    "triggerDeployTestContract": "ts-node ./script/index.ts triggerDeployTestContract",
+    "api404Test": "ts-node ./script/index.ts api404Test"
```
Comment (rafaelcr) [inline package.json:16]: For consistency, you could change these names to follow the package.json standards: `test:anchor-block`, `test:deploy-contract`, etc...

**Comment by rafaelcr** [comment]
@timstackblock as a low priority change, you should consider using pino for logs (instead of `console.log`) so we can get more info about the process and error stack etc. in logs: https://github.com/pinojs/pino

File: .env (hunk 1 above)
Comment (timstackblock) [reply .env:3]: Ok will do

File: package.json (hunk 2 above)
Comment (timstackblock) [reply package.json:16]: ok will do

**Comment by timstackblock** [comment]
Ok thank you for the feed back I am not familiar with Pino logs, I will read up on pinojs and find out how to implement them.

**Pull request merged**